### PR TITLE
100% coverage

### PR DESCRIPTION
**Pull request by jo**
- [ ] update https://github.com/eHealthAfrica/kazana/issues/131 when merged

### CODE REVIEW

**Comment by jo** [comment]
Blocked by https://github.com/eHealthAfrica/kazana-server/pull/109

**Comment by gr2m** [comment]
https://github.com/eHealthAfrica/kazana-server/releases/tag/v3.9.6 released. I restarted the travis build, now getting `Cannot find module 'boom'` error

**Comment by jo** [comment]
Thanks @gr2m :+1: 
This is still WIP, sorry if my last comment was confusing. Will continue tomorrow.

**Comment by jo** [comment]
`[boom` was a devDependency](https://github.com/eHealthAfrica/kazana-raw-data/blob/master/package.json#L40) which I have removed when I replaced the mock server with a real one.

This would have caused a disfunctional deploy if we'd used `npm install --production`, which we do AFAIK. It reminds me that we should use the production flag in our integration tests.

**Comment by jo** [comment]
WIP: This still needs tests for most attachment api endpoints.

**Comment by jo** [comment]
This is ready now

**Comment by jo** [comment]
please wait, I'll try to reach 100% branch coverage

**Comment by jo** [comment]
done. Branch coverage is now at 100%.

**Comment by FrancescoK** [comment]
:clap: :clap: :clap:
:tada:

File: lib/routes/submissions/update.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ function updateSubmissionHandler (request, reply) {
   updateSubmissionDoc(report, user)
 
   var promise = user.isAdmin
+  // FIXME
+    // ? store.find(id)
```
Comment (gr2m) [inline lib/routes/submissions/update.js]: what’s this?

Comment (jo) [inline lib/routes/submissions/update.js]: oh, leftover. Will remove that comment and have a look

Comment (jo) [inline lib/routes/submissions/update.js]: I have removed that obsolete comment.

**Comment by gr2m** [comment]
very nice :+1:

**Comment by jo** [comment]
let me see what happens if I try to merge with pending checks...

**Comment by jo** [comment]
hmmm it worked.

**Comment by jo** [comment]
WTF?

**Comment by jo** [comment]
I was able to merge the pr regardless of the status checks:

screenshot from 2016-01-14 17-26-42

**Pull request merged**